src/components/icons/ColonyIcon.tsx

```diff
@@ -1,41 +1,101 @@
 import { type Icon, IconBase, type IconWeight } from '@phosphor-icons/react';
 import React, { forwardRef, type ReactElement } from 'react';
 
-const SVG = (
-  <>
-    <path
-      fill="#101828"
-      d="M128,64.7c17.7,0,32.1-14.5,32.1-32.3S145.7,0,128,0c-17.7,0-32.1,14.5-32.1,32.3
-		S110.3,64.7,128,64.7z"
-    />
-    <path
-      fill="#101828"
-      d="M128,256c17.7,0,32.1-14.5,32.1-32.3c0-17.9-14.4-32.3-32.1-32.3c-17.7,0-32.1,14.5-32.1,32.3
-		C95.9,241.5,110.3,256,128,256z"
-    />
-    <path
-      fill="#101828"
-      d="M222.7,160.3c17.7,0,32.1-14.5,32.1-32.3c0-17.9-14.4-32.3-32.1-32.3
-		c-17.7,0-32.1,14.5-32.1,32.3C190.6,145.9,204.9,160.3,222.7,160.3z"
-    />
-    <path
-      fill="#101828"
-      d="M33,160.3c17.7,0,32.1-14.5,32.1-32.3S50.8,95.7,33,95.7S0.9,110.1,0.9,128S15.3,160.3,33,160.3
-		z"
-    />
-    <path fill="#101828" d="M128.1,76L76.5,128l51.6,51.9l51.6-51.9L128.1,76z" />
-    <path fill="#101828" d="M185.9,69.7V14.1l55.2,55.6H185.9z" />
-    <path fill="#101828" d="M69.8,69.7H14.6l55.2-55.6V69.7z" />
-    <path fill="#101828" d="M69.8,186.3v55.6l-55.2-55.6H69.8z" />
-    <path fill="#101828" d="M185.9,186.3h55.2l-55.2,55.6V186.3z" />
-  </>
-);
+import { usePageThemeContext } from '~context/PageThemeContext/PageThemeContext.ts';
 
-const weights = new Map<IconWeight, ReactElement>([['regular', SVG]]);
+const useGetIcon = () => {
+  const { isDarkMode } = usePageThemeContext();
 
-const ColonyIcon: Icon = forwardRef((props, ref) => (
-  <IconBase ref={ref} {...props} weights={weights} />
-));
+  return !isDarkMode ? (
+    <>
+      <path
+        fill="#101828"
```

Comment:
Instead of hardcoding the `fill` value for each of the `path` elements and using a ternary inside this component, I think it will be better if you set all `fill` property values to `currentColor` so like:

```js
<path
  fill="currentColor"
  d="M128,256c17.7,0,32.1-14.5,32.1-32.3c0-17.9-14.4-32.3-32.1-32.3c-17.7,0-32.1,14.5-32.1,32.3
    	C95.9,241.5,110.3,256,128,256z"
/>;
```

Then in usage:

```js
<ColonyIcon className={clsx('whatever-classes', {
  'text-gray-900': Theme1,
  'text-pink-xxx': Theme2,
  'text-whatever': ThemeWhatever
})}  />
```

Reply:
This is what I did in my new navigation PR: https://github.com/JoinColony/colonyCDapp/blob/237ced1395e0e6d2cd81ff571e2b010e46906c0e/src/components/icons/ColonyIcon.tsx

Reply:
Hey @CzarekDryl I'm gonna leave this code snippet here. Apologies for being slightly invasive regarding implementation suggestions but I thought it's best to discuss this since I will be using this Icon in my PR as well. It will likely be rendered in areas where the container's colour is the same as the light-themed Icon background. So I think it's good to have it configurable through props.

I want it known that if there's an issue with this, I will 100% take the blame!! 😂 

You can write the `ColonyIcon` as:

```ts
import {
  type Icon,
  IconBase,
  type IconWeight,
  type IconProps,
} from '@phosphor-icons/react';
import clsx from 'clsx';
import React, { forwardRef, type ReactElement } from 'react';

const SVG = () => (
  <svg
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 38 38"
    className="h-full w-full"
  >
    <g>
      <path
        fill="transparent"
        d="M19 38c10.493 0 19-8.507 19-19S29.493 0 19 0 0 8.507 0 19s8.507 19 19 19Z"
      />
      <path
        fill="currentColor"
        d="M19 13.042a3.065 3.065 0 1 0 0-6.13 3.065 3.065 0 0 0 0 6.13ZM19 31.09a3.065 3.065 0 1 0 0-6.129 3.065 3.065 0 0 0 0 6.13ZM28.023 22.065a3.065 3.065 0 1 0 0-6.13 3.065 3.065 0 0 0 0 6.13ZM9.977 22.065a3.065 3.065 0 1 0 0-6.13 3.065 3.065 0 0 0 0 6.13ZM23.904 19.004 19 14.1l-4.903 4.904L19 23.908l4.904-4.904ZM24.516 13.485V8.237l5.248 5.248h-5.248ZM13.485 13.485H8.237l5.248-5.248v5.248ZM13.485 24.519v5.248l-5.248-5.248h5.248ZM24.516 24.516h5.248l-5.248 5.248v-5.248Z"
      />
    </g>
  </svg>
);

const ColonyIcon: Icon = forwardRef<SVGSVGElement, IconProps>(
  ({ className, ...rest }, ref) => {
    const weights = new Map<IconWeight, ReactElement>([['regular', SVG()]]);

    return (
      <div className={clsx(className, 'overflow-hidden rounded-full')}>
        <IconBase ref={ref} {...rest} weights={weights} />
      </div>
    );
  },
);

ColonyIcon.displayName = 'ColonyIcon';

export default ColonyIcon;
```

Then usage:

```ts
const { isDarkMode } = usePageThemeContext();

<ColonyIcon
  size={36}
  className={clsx({
    'bg-[#00284B] text-base-white': isDarkMode,
    'bg-base-white text-[#00284B]': !isDarkMode,
  })}
/>
```

Please let me know if this works on your end. I'm very open to suggestions as well!

Reply:
I've just followed up with Mel, it seems that my assumption is correct that we're not supposed to swap between different SVG variations so we can still maintain the ratio. But because I'll be working on this anyway, you can leave your implementation as it is @CzarekDryl 👌

Reply:
@CzarekDryl @rumzledz The screenshot above from the design system is for CLYN token avatars and should not be used at all in the navigation areas. The stand alone Colony logo should be used and exported from Figma files. Please don't use these token avatars as they are just for dummy content in the Figma designs.

Reply:
Additionally, if you need help finding the Colony logo, please let me know.